TagFilterModule/tag_filter.py

```diff
@@ -0,0 +1,195 @@
+
+        # arguments available for the user to customize
+        parser.add_argument('APIkey', type = str, help = "Your API Key")
+        parser.add_argument('name', type = str, help = 'Name of the Tag')
+        parser.add_argument('-c', '--csv', type = str, help = "Name of CSV File", default = 'tags')
+        parser.add_argument('-m', '--movement', type = str, help = 'Filter by the Type of Movement')
+        parser.add_argument('-l', '--limit', type = int, help = "Limit the number of movements tallied")
+        parser.add_argument('-s', '--startTime', type = str, help = 'Start time fo data collection yyyy-mm-dd (0)0:00:00')
+        parser.add_argument('-e', '--endTime', type = str, help = "End time of data collection yyyy-mm-dd (0)0:00:00")
+        return parser
+
+    # return unique values of a list
+    def unique(self, list):
+        unique_lst = []
+        for value in list:
+            if value not in unique_lst:
+                unique_lst.append(value)
+        return unique_lst
+
+    # converts the timestamp to ms time
+    def milliseconds_time(self, human):
+        # the +25200000 is to produce local time and not GMT
+        ms_time = (calendar.timegm(time.strptime(human, '%Y-%m-%d %H:%M:%S')) * 1000) + 25200000
+        return ms_time 
+
+    # converts the ms time to timestamp
+    def human_time(self, event):
+        event = event/1000
+        timestamp = time.strftime("%a, %d %b %Y %H:%M:%S", time.localtime(event))
+        return timestamp
+
+    # converts the tag uuid to tag name
+    def tag_uuid_convert(self, uuid, tag_name_data):  
+        for value in tag_name_data['proximityStates']:
+            if uuid == value['tagUuid']:
+                return value['name']
+
+    # converts the tag name to tag uuid
+    def tag_name_convert(self):    
+        for value in self.tag_name_data['proximityStates']:
+            if self.args.name == value['name']:
+                return value['tagUuid']
+
+    # returns data with device uuid and device name
+    def tag_name(self): 
+            endpoint = self.api_url + '/api/proximity/getMinimalProximityStateList'
+            payload = {
+            }
+            resp = self.api_sess.post(endpoint, json=payload,
+                            verify=False)
+            order_content = resp.content.decode('utf8')
+            tag_name_data = json.loads(order_content)
+            return tag_name_data
+
+    # returns data with address
+    def locations(self):
+        endpoint = self.api_url + "/api/location/getLocations"
+        payload = {
+        }
+        resp = self.api_sess.post(endpoint, json=payload,
+                       verify=False)
+        order_content = resp.content.decode('utf8')
+        # Load the JSON to a Python list & dump it back out as formatted JSON
+        location_data = json.loads(order_content)
+        return location_data
+
+    # converts lat and lon to street address
+    def location_data_print(self, location_data, lat_1):
```

Comment:
I don't think this does quite what you are expecting.  By comparing just the latitude, I can theoretically have something on the other side of the world be associated into this location.